crt_portal/cts_forms/templates/forms/complaint_view/index/filters/date.html

```diff
@@ -0,0 +1,17 @@
+{% extends "forms/complaint_view/index/filter.html" %}
+{% block controls %}date{% endblock %}
+{% block label %}Date{% endblock %}
+{% block id %}date{% endblock %}
+
+{% block content %}
+<div class="usa-date-range-picker">
+      <label class="usa-label margin-bottom-1" for="date_range_filter"><b>Submission Date</b></label>
+      <div class="usa-date-picker">
+        <label for="id_{{form.create_date_start.name}}" class="usa-label" for="{{form.create_date_start.name}}">From:</label>
+        {{ form.create_date_start }}
+        <label for="id_{{form.create_date_end.name}}" class="usa-label" for="{{form.create_date_end.name}}">To:</label>
+        {{ form.create_date_end }}
+        <button class="usa-button margin-top-2" type="submit" id="id_apply_dates">Apply Dates</button>
+      </div>
```

Comment:
This may be too much for this PR right now but we do have the `get_field_label` template tag which we can use to override the field label. Having `Submission start date: 2020-08-05` looks better than `create date start: 2020-08-05` imo. We use this template tag to change, for instance, `assigned_to` to `Assignee`. Anyway, just noting this for the future, maybe we can have a follow up task for this.

Reply:
Sounds good.